src/steps/11-GeneratePackageDocuments/UploadJAMRRDocuments.vue

```diff
@@ -140,12 +140,13 @@ import {AttachmentServiceCallbacks} from "@/services/attachment";
 export default class UploadJAMRRDocuments extends Mixins(SaveOnLeave) {
   private fairOppDTO = AcquisitionPackage.getInitialFairOpportunity();
 
-  private attachmentServiceName = FAIR_OPPORTUNITY_TABLE;
+  private attachmentServiceName = ACQUISITION_PACKAGE_TABLE;
   private maxFileSizeInBytes = 1073741824;
   private invalidFiles: invalidFile[] = [];
-  private validFileFormats = ["pdf","jpg","png"];
+  private validFileFormats = ["pdf","jpg","png","docx"];
```

Comment:
add another item here - `private restrictedNames = [...` and then the array of file names to prevent being used. Pass `restrictedNames` down to `ATATFileUpload` as a prop (around line 96 or so)